src/modules/payload_deliverer/gripper.h

```diff
@@ -99,9 +102,17 @@ class Gripper
 	// Returns true if in released position either sensed or timeout based
 	bool released() { return _state == GripperState::RELEASED; }
 
-	// Returns true only once after the state transitioned into released
-	// Used for sending vehicle command ack only when the gripper actually releases
-	// in payload deliverer
+	/**
+	 * @brief Returns true once after gripper succeededs in releasing the payload
```

Comment:
Quick note here: `@brief` should usually be used to describe the function purpose. Usually you use the `@return` tag to identify what the function returns.